Felienne/hedy
# [Cleanup] Removed language dependency on url argument

**Description**
Currently we depend on an `lang` argument added to the url to tell us the language we have to render the current page in. This is cumbersome as we have to pre-process each url to make sure the user keeps the same language when switching pages. In this PR we implement the chosen language in a session variable, living on the server. Removing the need from sending the argument with the url and in the same process cleaning this url as well. This PR will also make it significant easier to implement #1478.

**Fix for**
This PR fixes #1496.

**How to test**
Navigate through different pages and notice that the `lang` attribute is no longer passed in the url. Try different languages to verify the correct updating of the stored session variable.

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
